src/Context/ContextKeys.php

```diff
@@ -16,6 +16,13 @@ public static function span(): ContextKeyInterface
         return $instance ??= Context::createKey('opentelemetry-trace-span-key');
     }
 
+    public static function localRootSpan(): ContextKeyInterface
+    {
+        static $instance;
+
+        return $instance ??= Context::createKey('opentelemetry-trace-local-root-span-key');
+    }
```

Comment:
Should live in the API package, not in the context package.

The span and baggage key were moved to the context package to allow special casing them in `Context::with()` to avoid copying the internal `$context` array (currently only done for spans). The local root span won't be updated as frequently -> does not need this optimization.
https://github.com/open-telemetry/opentelemetry-php/blob/4f3281706545bf1efcb9962a3f9617fb43c585b1/src/Context/Context.php#L105-L109